src/vm/docs/mod.rs

```diff
@@ -84,6 +84,20 @@ contract principal.",
     example: "(print tx-sender) ;; Will print out a Stacks address of the transaction sender",
 };
 
+const TOTAL_LIQUID_USTX_KEYWORD: KeywordAPI = KeywordAPI {
+    name: "total-liquid-ustx",
+    output_type: "uint",
+    description: "Returns the total number of micro-STX (uSTX) that are liquid in the system as of this block.",
+    example: "(print total-liquid-ustx) ;; Will print out the total number of liquid uSTX"
+};
+
+const REGTEST_KEYWORD: KeywordAPI = KeywordAPI {
+    name: "is-in-regtest",
+    output_type: "bool",
+    description: "Returns whether or not the code is running in a regression test",
+    example: "(print is-in-regtest) ;; Will print 'true' if the code is running in a regression test"
```

Comment:
Should we expose the `chain-id` instead?

Reply:
We don't have a regtest chain ID, as far as I know.  The value `0x80000000` we use in most unit tests is just a placeholder.

Reply:
@kantai do you have a take on this one?